CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 4.2.0
+
+This is a major Teleport release with a focus on new features and bug fixes.
+
+### Improvements
+
+* Alpha: Enhanced Session Recording let's you know what's really happening during a Teleport Session. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/features/enhanced_session_recording/).
+* Alpha: Workflows API lets admins escalate RBAC roles. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/enterprise/#approval-workflows).
+* Beta: Teleport provides HA Support using Firestone and Google Cloud Storage using Google Cloud Platform. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/gcp_guide/).
+* Remote tctl execution is now possible. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/cli-docs/#tctl). [#1525](https://github.com/gravitational/teleport/issues/1525) [#2991](https://github.com/gravitational/teleport/issues/2991)
```

Comment:
Is this sufficiently documented?  I see reference to `--auth-server` and `--identity` for `tctl auth sign` and `tctl status` but there is no reference of these flags for other `tctl` commands or in the global flags?

Reply:
Yeah - I suspect there are a lot more instances where these flags could be used. We can push this for now though and update the docs separately later.

Reply:
Yes, this was a last min fix by me. I'll add a section into the admin manual for reasons why we've built this feature and How to use it.